website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -662,7 +662,7 @@ Revisit step 9 of the [upgrade procedure]({{< ref "#upgrading" >}}) and ensure t
   * AMI Selector Terms has a new Alias field which can only be set by itself in `EC2NodeClass.Spec.AMISelectorTerms`
```

Comment:
Can you make this change in the `preview` version of the docs as well? Happy to take this but the `preview` one is what gets carried forward -- also, could probably just apply this to all versions!